DistributedCircuitbreaker.Redis/DependencyInjection/CircuitBreakerRedisCollectionExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class CircuitBreakerRedisCollectionExtensions
             if(redisOptions.RedisConnectionConfiguration == null)
                 throw new ArgumentNullException(nameof(redisOptions.RedisConnectionConfiguration));
 
-            collection.AddDistributedRedisCache(options => options.Configuration = redisOptions.RedisConnectionConfiguration);
+            collection.AddStackExchangeRedisCache(options => options.Configuration = redisOptions.RedisConnectionConfiguration);
```

Comment:
Could you add more details to the PR description? Did you consider creating another extension instead of changing this one?